python/doc/examples/probabilistic_modeling/distributions/plot_conditional_distribution.py

```diff
@@ -10,41 +10,58 @@
 
 
 # %%
-# In this example we are going to build the distribution of the random vector :math:`\vect{X}` conditioned by the random vector :math:`\vect{\Theta}`
+# In this example we are going to build the distribution of the random vector :math:`\inputRV`
+# defined by the conditional distribution of:
 #
 # .. math::
-#    \vect{X}|\vect{\Theta}
 #
-# with :math:`\vect{\Theta}` obtained with the random variable :math:`Y` through a function :math:`f`
+#    \inputRV|\vect{\Theta}
+#
+# where :math:`\vect{\Theta}` is the output of the random variable :math:`\vect{Y}` through the link
+# function :math:`f`:
 #
 # .. math::
-#    \vect{\Theta}=f(Y)
+#
+#    \vect{\Theta} & = f(\vect{Y})\\
+#    \vect{Y} & \sim \cL_{\vect{Y}}
+#
+# This example creates a :class:`~openturns.DeconditionedDistribution` which offers all the methods
+# attached to the distributions.
+#
+# We consider the case where :math:`X` is of dimension 1 and follows a uniform distribution defined
+# by:
+#
+# .. math::
+#
+#    X|(A,B) & \sim \cU(Y, 1+Y^2) \\
```

Comment:
There is no universal notation for each distribution: I suggest to explain what means the caligraphic U.

Reply:
yes